app/content/deployment-options/custom-deployment/release-notes.md

```diff
@@ -0,0 +1,11 @@
+## Release Notes
+
+### 21.08.19-2
+* PostgreSQL Data Base upgraded to version 12.7
+* Ability to disable Diagnosis endpoint
+* Fix ArcGIS import issue when there are no features
+* Support for latest HERE API key format
+* General improvements and performance optimizations
+
+### 21.06.14
+* First stable version
```

Comment:
This is not the first stable version for the on-prem. We should add more detail like "First stable version based on Docker".

Reply:
Clarification added!

Thanks!